src/NuGetForUnity/Editor/NugetConfigFile.cs

```diff
@@ -74,6 +74,37 @@ public class NugetConfigFile
         /// </summary>
         public bool ReadOnlyPackageFiles { get; set; }
 
+        public string PackagesConfigPath;
```

Comment:
```suggestion
        public string PackagesConfigPath { get; set; }
```
Using public fields is a bad practice, use properties instead.